### ISSUE TITLE
Cpptraj header title - inconsistent 

### ISSUE THREAD

**Issue by hainm** [Amber-MD/cpptraj#372]
`generated` vs `Generated`

``` bash
 $ grep "Cpptraj generated" *
Action_CreateReservoir.cpp:    title.assign("Cpptraj generated structure reservoir");
Binary file Action_CreateReservoir.o matches
Traj_CharmmDcd.cpp:      SetTitle("Cpptraj generated dcd file.");
Binary file Traj_CharmmDcd.o matches
Traj_Mol2File.cpp:    SetTitle("Cpptraj generated mol2 file.");
Binary file Traj_Mol2File.o matches
Traj_SQM.cpp:    outTitle.assign("Cpptraj generated SQM input");
Binary file Traj_SQM.o matches
Traj_Tinker.cpp:    SetTitle("Cpptraj generated mol2 file.");
```

``` bash
$ grep "Cpptraj Gen" *
Traj_AmberCoord.cpp:      title.assign("Cpptraj Generated trajectory");
Binary file Traj_AmberCoord.o matches
Traj_AmberNetcdf.cpp:      SetTitle("Cpptraj Generated trajectory");
Binary file Traj_AmberNetcdf.o matches
Traj_AmberRestart.cpp:    outTitle.assign("Cpptraj Generated Restart");
Traj_AmberRestartNC.cpp:    SetTitle("Cpptraj Generated Restart");
Binary file Traj_AmberRestartNC.o matches
Binary file Traj_AmberRestart.o matches
Traj_NcEnsemble.cpp:      SetTitle("Cpptraj Generated trajectory");
```

**Comment by hainm** [comment]
I raise this issue because I use those header to make sure `pytraj` writes to correct format.

**Comment by drroe** [comment]
Can you just do a case-insensitive check? If it's a pain I can change the titles to be something more consistent, but this may avoid any similar issues in the future.

**Comment by hainm** [comment]
> Can you just do a case-insensitive check

Yes.

>  If it's a pain I can change the titles to be something more consistent,

No, it's not a pain. Just a minor point for consistency.

**Comment by drroe** [comment]
Addressed by #648
